packages/user-ui/src/pages/projects/_id/components/basic_info/controllers.rs

```diff
@@ -0,0 +1,73 @@
+use bdk::prelude::*;
+use models::{DeliberationBasicInfo, DeliberationBasicInfoQuery, DeliberationBasicInfoSummary};
+
+#[derive(Debug, Clone, Copy, DioxusController)]
+pub struct Controller {
+    #[allow(dead_code)]
+    lang: Language,
+    #[allow(dead_code)]
+    deliberation_id: ReadOnlySignal<i64>,
+
+    basic_info: Resource<DeliberationBasicInfoSummary>,
+}
+
+impl Controller {
+    pub fn new(
+        lang: Language,
+        deliberation_id: ReadOnlySignal<i64>,
+    ) -> std::result::Result<Self, RenderError> {
+        let basic_info = use_server_future(move || {
+            let deliberation_id = deliberation_id();
+            async move {
+                let res = DeliberationBasicInfo::get_client(&crate::config::get().api_url)
+                    .query(
+                        deliberation_id,
+                        DeliberationBasicInfoQuery::new(1).with_page(1),
+                    )
+                    .await;
+                match res {
+                    Ok(v) => {
+                        if v.total_count == 1 {
+                            v.items[0].clone()
+                        } else {
+                            DeliberationBasicInfoSummary::default()
+                        }
+                    }
+                    _ => DeliberationBasicInfoSummary::default(),
+                }
+            }
+        })?;
+
+        let ctrl = Self {
+            lang,
+            deliberation_id,
+            basic_info,
+        };
+
+        Ok(ctrl)
+    }
+    #[allow(unused)]
```

Comment:
```suggestion
    #[cfg(feature = "web")]
```
여기에서 이렇게 정의하는게 나을지도 모릅니다.
아마 unused 가 제 생각에는 web 일때만 사용해서 나오는거인거 같아서요.